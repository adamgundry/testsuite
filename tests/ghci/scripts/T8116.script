:m + Data.Complex
((-2) :+ 0)**(1.5 :+ 0)
