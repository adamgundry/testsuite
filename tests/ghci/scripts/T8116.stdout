(-5.195736337412959e-16) :+ (-2.82842712474619)
